mappings/net/minecraft/util/profiling/jfr/JfrProfiler.mapping

```diff
@@ -6,13 +6,35 @@ CLASS net/minecraft/class_6505 net/minecraft/util/profiling/jfr/JfrProfiler
 	FIELD field_34407 EVENTS Ljava/util/List;
 	FIELD field_34408 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_34409 CONFIG_PATH Ljava/lang/String;
+	FIELD field_34410 DATE_TIME_FORMAT Ljava/time/format/DateTimeFormatter;
 	FIELD field_34411 currentRecording Ljdk/jfr/Recording;
 	METHOD method_37980 stop ()Ljava/nio/file/Path;
 	METHOD method_37981 start (Lnet/minecraft/class_6505$class_6506;)Z
 		ARG 0 instanceType
+	METHOD method_37982 (Lnet/minecraft/class_6505$class_6506;Ljava/lang/String;Ljdk/jfr/Recording;)V
+		ARG 2 recording
 	METHOD method_37983 start (Ljava/io/Reader;Lnet/minecraft/class_6505$class_6506;)Z
+		ARG 0 reader
 		ARG 1 instanceType
 	METHOD method_37984 isProfiling ()Z
 	CLASS class_6506 InstanceType
+		FIELD field_34414 type Ljava/lang/String;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
+			ARG 3 type
+		METHOD method_37986 getType ()Ljava/lang/String;
```

Comment:
```suggestion
		FIELD field_34414 name Ljava/lang/String;
		METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
			ARG 3 name
		METHOD method_37986 getName ()Ljava/lang/String;
```
Duplicating the name (i.e. `type` in `instanceType.getType()`) should be avoided.